src/ChannelCreator.php

```diff
@@ -4,20 +4,26 @@
 
 namespace Stock2Shop\Connector;
 
-use Stock2Shop\Connector\Config\Environment;
-use Stock2Shop\Connector\Config\LoaderDotenv;
+use Stock2Shop\Environment\LoaderDotenv;
+use Stock2Shop\Environment\Env;
 use Stock2Shop\Share;
+use Stock2Shop\Share\Channel\ChannelOrdersInterface;
 
 class ChannelCreator extends Share\Channel\ChannelCreator
 {
     public function __construct()
     {
         $loader = new LoaderDotenv(__DIR__ . '/../');
-        Environment::set($loader);
+        Env::set($loader);
```

Comment:
Constructor should not load env vars.
This package is intended to be loaded by another application which should set env vars